templates/default/haproxy.cfg.erb

```diff
@@ -70,7 +70,9 @@ defaults
 frontend <%= frontend %>
   default_backend <%= f['default_backend'] %>
 <% unless f['bind'].empty? -%>
```

Comment:
Mmh sorry @damacus, I don't understand.
Should I remove control as it is already a mandatory value (for chef)?